applications/fax/src/fax.hrl

```diff
@@ -79,40 +79,5 @@
 
 -define(PORT, kapps_config:get_integer(?CONFIG_CAT, <<"port">>, 30950)).
 
--define(DEFAULT_CONVERT_PDF_CMD
```

Comment:
remove:

* `FAX_EXTENSION`
* `OPENOFFICE_COMPATIBLE`
* `OPENOFFICE_MIME_PREFIX`
* `OPENXML_MIME_PREFIX`

Reply:
OPENOFFICE_MIME_PREFIX and OPENXML_MIME_PREFIX are required for other things in fax.hrl like DEFAULT_ALLOWED_CONTENT_TYPES, should I import these from kz_convert.hrl